app/boards/shields/romac_plus/boards/nice_nano.overlay

```diff
@@ -0,0 +1,28 @@
+&spi1 {

```

Comment:
Can you add a license header here too? Thanks.

Reply:
Updated as a part of [4c09204](https://github.com/zmkfirmware/zmk/pull/198/commits/4c092044ce22f0c60c3bf0ef860ecc543328d5bf)